### PR TITLE
added cleaning rice

### PR DESCRIPTION
**Pull request by lightme-fan**
Hi everyone, 

could you look at my PR, I add one file, I am happy if there is anything to be changed

Thank in advance

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# chores
+git# chores
```
Comment (bakozetra) [inline README.md]: oops this was not meant to happen

File: cleaning rice.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Cleaning rice
```
Comment (bakozetra) [inline cleaning rice.md]: This file name should not have spaces

**Review by bakozetra — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lightme-fan) [reply README.md]: Thank you

**Comment by lightme-fan** [comment]
closed this because Kati's pull request

**Pull request closed without merging**